code/main.cpp

```diff
@@ -1,35 +1,41 @@
-/*
- * This main is used to test the canbus or beetle class.
- */
+
 #include <beetle.hpp>
 #include <comm.hpp>
 #include <hardware_usart.hpp>
 #include <hwlib.hpp>
+#include <rhino.hpp>
 
 int main(void) {
     // kill the watchdog
     WDT->WDT_MR = WDT_MR_WDDIS;
     hwlib::wait_ms(1000);
 
-    // Create QIK board that handles the motors.
+    // Create two encode pins for the motors.
     auto encode_m0 = hwlib::target::pin_adc(hwlib::target::ad_pins::a0);
     auto encode_m1 = hwlib::target::pin_adc(hwlib::target::ad_pins::a1);
+
+    // Create reset pin for the qik board
     auto qik_2s12v10_reset_pin = hwlib::target::pin_out(2, 25); // digital pin 5
-    auto usart = r2d2::usart::hardware_usart_c<r2d2::usart::usart0>(9600);
+
+    // Create usart object.
+        auto usart = r2d2::usart::hardware_usart_c<r2d2::usart::usart0>(115200);
```

Comment:
This line's indent is incorrect

Reply:
https://github.com/R2D2-2019/moving_platform/pull/30/commits/828469270908ab8c8bdaf7bc050cf363f1299915